### ISSUE TITLE
Feature request: suspend/set quiet for a certain amount of time

### ISSUE THREAD

**Issue by blahgeek** [rcaelers/workrave#305]
(Firstly, thanks for maintaining this incredible app, it has been very helpful!)

I sometimes will switch workrave to "suspended" or "quiet" mode when I don't want any interruption (e.g. doing a presentation or screen sharing at work). However, I always forget to reset it to normal after that, it may take me half a day to realize that it's still suspended.

I think it would be great if, in the context menu, there's additional options that allows us to set to "suspended" or "quiet" mode for certain amount of time (e.g. 1hour) and automatically reset to normal mode after that.

**Comment by phgachoud** [comment]
I would add a shortcut on the main window for that too, as well as the 'set status' with its options

**Comment by freetimecoder3** [comment]
I'd like the feature too.   And thanks for the gr8 app.

**Comment by burbilog** [comment]
This was asked in 2008, I remember posting in that ticket in 2011: https://workrave.org/bugzilla/712

Yes, it's a huge problem for me -- too often I suspend workrave to do some work during important calls in the morning and discover myself in pain eight hours later because workrave is still suspened...

I added this line to crontab:
~~~
DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/1000/bus
0 7 * * * dbus-send --type=method_call --dest=org.workrave.Workrave /org/workrave/Workrave/Core org.workrave.CoreInterface.SetOperationMode string:'normal
~~~

And it does re-enable workrave every morning. Alas, can't do that on hourly basis, because sometimes I have to participate in really long and important talks.

**Comment by blipp** [comment]
Duplicate of #98 ?

**Comment by Chealer** [comment]
Not exactly, but it does look like the solution for #98 will also solve this one.

**Comment by wolftune** [comment]
Looks like a duplicate to me. Having "quiet" and "suspend" each with a "for X time" function seems basically like the request. I'd suggest consolidating the two tickets

**Comment by rcaelers** [comment]
This has been implemented in 1.11, which is now in beta
